daily_practice/challenges/objects-challenges.test.js

```diff
@@ -159,14 +160,15 @@ let pokeTwo = {
     pokemon_type: "Water"
 }
 // Write a function called describePokemon() that take an object like the ones above and uses destructuring to return a description of the Pokemon such that:
-var { species, pokemon_type } = pokeOne
-var { species, pokemon_type } = pokeTwo
-
-function describePokemon(object){
 
-  return `${object.this.species} is a ${object.this.pokemon_Type} pokemon`
+function describePokemon1(pokeOne){
```

Comment:
You do not need to define two separate function here. The same function works for both

Reply:
The parameter name should be something like pokemon, you do not need to make it match the variable name that you are wanting to pass in. Seems like we need to talk more about functions in our next chat.